src/profile/components/editProfileForm/EditProfileForm.tsx

```diff
@@ -406,25 +422,61 @@ function EditProfileForm(props: Props) {
                           <Field
                             className={styles.formField}
                             as={TextInput}
+                            maxLength="128"
                             name={`addresses.${index}.address`}
                             id={`addresses.${index}.address`}
                             labelText={t('profileForm.address')}
+                            invalid={getIsInvalid(
+                              formikProps,
+                              `addresses.${index}.address`
+                            )}
+                            helperText={getFieldError(
+                              formikProps,
+                              `addresses.${index}.address`,
+                              {
+                                max: 128,
+                              }
+                            )}
                           />
 
                           <Field
                             className={styles.formField}
+                            maxLength="5"
```

Comment:
Is it so that this covers all the postal codes in all the world's countries?

Reply:
I think the improved postal code validation will be added later. Idea for now is that it just does the same thing as primary addresses postal code field.